.github/workflows/update-pulp-packages.yml

```diff
@@ -16,15 +16,17 @@ jobs:
     name: "Generate Package List"
     runs-on: ubuntu-latest
     if: github.event_name == 'push' && github.head_ref == 'rpm/develop'
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
     steps:
     - name: Sort Updated Libs
       if: github.event_name == 'push' && github.head_ref == 'rpm/develop'
       run: git diff --unified=0 ${{github.event.before}} ${{github.sha}} automation/requirements.txt | grep -Po '(?<=^\+)(?!\+\+).*' > $PACKAGE_TO_UPDATE
     - name: Parse Package List
       run: ./build_matrix.py
     - name: Load Packages Matrix
-      run: |
-        echo "matrix=$(cat packages.json)" >> $GITHUB_ENV
+      id: set-matrix
+      run: echo "matrix=$(jq -c . < ./packages.json)" >> $GITHUB_OUTPUT
```

Comment:
Why is it needed to use `jq`? Wouldn't it be better to modify `build_matrix.py` to generate compact output?